src/transactions/feed/detailContent/JumpstartReclaimBottomSheet.tsx

```diff
@@ -0,0 +1,68 @@
+import { BottomSheetScreenProps } from '@th3rdwave/react-navigation-bottom-sheet'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text } from 'react-native'
+import { Colors } from 'react-native/Libraries/NewAppScreen'
+import BottomSheetScrollView from 'src/components/BottomSheetScrollView'
+import Button, { BtnSizes } from 'src/components/Button'
+import DataFieldWithCopy from 'src/components/DataFieldWithCopy'
+import Logo from 'src/icons/Logo'
+import { navigateBack, navigateHome } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import Logger from 'src/utils/Logger'
+import EstimatedNetworkFee from 'src/walletConnect/screens/EstimatedNetworkFee'
+
+type Props = BottomSheetScreenProps<StackParamList, Screens.JumpstartReclaimBottomSheet>
+
+const TAG = 'JumpstartReclaimBottomSheet'
+
+function JumpstartReclaimBottomSheet({
+  route: {
+    params: { reclaimTx, networkId, tokenAmount },
+  },
+}: Props) {
+  const { t } = useTranslation()
+
+  const transactionString = JSON.stringify(reclaimTx)
+
+  async function onConfirm() {
```

Comment:
we haven't been terribly consistent about this but i think it is a bad pattern to trigger async functions on button presses because then we don't handle rejected promises well (and don't necessarily await the outcome before moving on). i started using the useAsyncCallback hook to handle this, maybe we can do something similar here? it would also enable the loading state out of the box. https://github.com/valora-inc/wallet/blob/0af2960d9bb876cfdd9ade50b1ab210fcd911345/src/jumpstart/JumpstartEnterAmount.tsx#L68